src/BlazorUI/Bit.BlazorUI.Extras/Components/PdfViewer/BitPdfViewer.scss

```
@@ -1,11 +1,16 @@
-@import "../../Styles/extra-variables.scss";
+﻿@import "../../Styles/extra-variables.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)